test/setup_transfer.cpp

```diff
@@ -252,36 +253,38 @@ void save_file(char const* filename, char const* data, int size)
 		fprintf(stderr, "ERROR writing file '%s': %s\n", filename, ec.message().c_str());
 		return;
 	}
-
 }
 
 bool print_alerts(lt::session& ses, char const* name
 	, bool allow_disconnects, bool allow_no_torrents, bool allow_failed_fastresume
-	, boost::function<bool(libtorrent::alert const*)> predicate, bool no_output)
+	, boost::function<bool(libtorrent::alert const*)> const& predicate, bool no_output)
```

Comment:
this will prevent ``predicate`` from being moved into this function. was that intentional?

Reply:
Yes, I intended it. C++98 doesn't have move semantics.